object_manager.go

```diff
@@ -23,7 +23,7 @@ type IBObjectManager interface {
 	CreateNetworkContainer(netview string, cidr string, isIPv6 bool, comment string, eas EA) (*NetworkContainer, error)
 	CreateNetworkView(name string, comment string, setEas EA) (*NetworkView, error)
 	CreatePTRRecord(networkView string, dnsView string, ptrdname string, recordName string, cidr string, ipAddr string, useTtl bool, ttl uint32, comment string, eas EA) (*RecordPTR, error)
-	CreateTXTRecord(recordname string, text string, ttl uint, dnsview string) (*RecordTXT, error)
+	CreateTXTRecord(recordName string, text string, dnsView string, useTtl bool, ttl uint32, comment string, eas EA) (*RecordTXT, error)
```

Comment:
Please take a look at CreateARecord() and put the arguments in a similar order.

Reply:
Updated so arguments for TXT record match the A Record order throughout the code base